sdk/src/main/java/com/kin/ecosystem/marketplace/presenter/MarketplacePresenter.kt

```diff
@@ -0,0 +1,381 @@
+package com.kin.ecosystem.marketplace.presenter
+
+
```

Comment:
whats the difference with the old file?

Reply:
removed  unused object and updated new logic related to new UI :)